lib/util/git/types.ts

```diff
@@ -0,0 +1,63 @@
+import type {
+  DiffResult as DiffResult_,
+  StatusResult as StatusResult_,
+} from 'simple-git';
+import type { GitOptions } from '../../types/git';
+
+export type StatusResult = StatusResult_;
+export type DiffResult = DiffResult_;
```

Comment:
Do we really need this? Can't we direct export those?

Reply:
Let's try...

Reply:
I think this is a leftover from `isolatedModules:true`, which was required for babel

Reply:
Yes, I don't know this story, but seemed pretty unnatural